packages/@sfx/sayt-driver-plugin/test/unit/common/index.test.ts

```diff
@@ -0,0 +1,9 @@
+import { expect } from '../../utils';
+import { SaytDriverPlugin as SaytDriverPluginExport} from '../../../src';
+import SaytDriverPlugin from '../../../src/sayt-driver-plugin';
+
+describe('Entry point', () => {
+  it('should export EventsBrowserPlugin ', () => {
+    expect(SaytDriverPluginExport).to.equal(SaytDriverPlugin);
+  });
+});
```

Comment:
I'm going to ignore the lack of interface exports for now because we decided to defer that decision.